spec/index.bs

```diff
@@ -1552,6 +1552,13 @@ A <dfn dfn>supported limits</dfn> object has a value for every limit defined by
         allowed in {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+    <tr><td><dfn>maxBindGroupsPlusVertexBuffers</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>4
```

Comment:
I just realized the default value here is 4, I think we meant to set it to 30 or 24 or something? I expect it to be higher than the defaults for maxBindGroups+maxVertexBuffers = 4+8.

@litherum did you have a proposed value for this?

Reply:
Editors: This should be 12 (maxBindGroups+maxVertexBuffers).

Reply:
On second thought, maybe it would be nicer if we set this to a higher limit, even though it's a little odd. I think both maxBindGroups=8 and maxVertexBuffers=16 are going to be common enough that it would be nice not to run into this limitation until you're pushing the boundaries a bit more. Maybe we can just set it to 24?

Unlike, for example, a case where `maxStorageBufferBindingSize > maxBufferSize`, I don't think there's much risk of applications programmatically using `maxBindGroupsPlusVertexBuffers - maxBindGroups` as a number of vertex buffers, or similar, that would mean we want to keep this value in a range that "makes sense". (The same could probably be said of the guarantee we added about `maxComputeInvocationsPerWorkgroup`.)

Reply:
Where did you get 24 from? 30 would probably be the most natural value, if we want something > 12

Reply:
It comes from 8+16 (common values of maxBindGroups+maxVertexBuffers), and leaves a generous amount of reserved space against Metal's 31.